spec/features/user_edits_a_song_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.feature "User edits a song" do
+  scenario "they see the page for the individual song" do
+    artist = create(:artist)
+
+    old_song_title = "One Love"
+    song_title = "Could You Be Loved"
+
+    visit artist_path(artist)
+    click_on "Edit"
+    fill_in "song_title", with: song_title
```

Comment:
"song_title" here is most likely not a good selector for the fill_in method. I like using the name attribute on the html. Try `fill_in '
song[title]'` instead